source/docs/en/reference.md

```diff
@@ -20,24 +58,24 @@ Our recommend Browser is Google Chrome.
 |Javascript|Express,SailsJS,JQuery,Bootstrap,AnglarJS,ReactJS|
 |SQL|SQLite|
 
-#### 3. APIs(Server) Challenges
-#### 4. HTML(Frontend) Challenges
-#### 5. Markdown Challenges
-#### 6. Mobile Challenges
-* Cordova
-* Android
-
-#### 7. Game Challenges
-* Unity
-* cocos2dx
-
-|Difficulty|Algorithm|Frameworks|APIs(Server)|HTML(Frontend)|Markdown|Mobile|Game|
-|:-:|---|---|---|---|---|---|---|
-|Very Hard|[Sudoku][sudoku]<br />[Rijndael][rijndael]|[EntityFramework][entity-framework]|[Event Application][eventapp]<br />[Login API][login-api]|||||
-|Hard||||||||
-|Normal|[Sudoku][sudoku-medium]<br />[Rijndael][rijndael-medium]|||||||
-|Easy|[Sudoku][sudoku-easy]<br>[array(C#)][arrays]<br />[ROT13][rot13]||[SQL][sql]|||||
-|Very Easy|[Fizzbuzz][fizzbuzz]<br />[Binary ToString][binary-tostring]||||||||
+##### API(Server) Challenges
+##### HTML(Frontend) Challenges
+##### Markdown Challenges
+##### Mobile Challenges
```

Comment:
Same idea as the other sample challenges, yes? I will also have this removed for the time being.